extras/docker/dave_noetic/Dockerfile

```diff
@@ -77,6 +76,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ros-${DIST}-rqt \
     ros-${DIST}-rqt-common-plugins \
     ros-${DIST}-geographic-msgs \
+
```

Comment:
Is this extra line intentional?

Reply:
Corrected in 5379d335e9783d1f97576045c97398be5fd9009f